SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -605,24 +606,54 @@ FRPCErrorInfo USpatialSender::SendRPC(const FPendingRPCParams& Params)
 {
 	SCOPE_CYCLE_COUNTER(STAT_SpatialSenderSendRPC);
 
+	bool bShouldDrop = false;
+
+	const float TimeDiff = (FDateTime::Now() - Params.Timestamp).GetTotalSeconds();
+	if (GetDefault<USpatialGDKSettings>()->QueuedOutgoingRPCWaitTime < TimeDiff)
+	{
+		bShouldDrop = true;
```

Comment:
Don't bother with should drop, just return immediately

Reply:
In that case logs will not contain the information about the TargetActor, Function etc. Is that acceptable?